atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CQLKeyValueService.java

```diff
@@ -389,9 +388,11 @@ private String getLocalDataCenter() {
             }
         }
         if (rowCount > fetchBatchCount) {
-            log.warn("Rebatched in getRows a call to " + tableRef.getQualifiedName() + " that attempted to multiget "
-                    + rowCount + " rows; this may indicate overly-large batching on a higher level.\n"
-                    + CassandraKeyValueServices.getFilteredStackTrace("com.palantir"));
+            log.warn("Rebatched in getRows a call to {} that attempted to multiget {} rows; "
```

Comment:
Have we spoken to Apollo team about how to log stack traces like this?  In particular, if we take a new line will they still be able to tell that the stack trace belongs with the log line?  In the past I've converted new lines in stack traces to another character so the stack trace is all on one line - this makes log processing much easier.